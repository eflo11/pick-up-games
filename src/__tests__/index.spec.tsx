import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Hello from '../';

it('Hello renders correctly', (): void => {
  const props = {
    compiler: 'newCompiler',
    framework: 'react',
  };
  const tree = renderer.create(<Hello {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
