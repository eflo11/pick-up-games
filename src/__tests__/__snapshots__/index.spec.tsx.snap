// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello renders correctly 1`] = `
<h2>
  Hello
   from 
  newCompiler
   and 
  react
  !
</h2>
`;
